src/main.cc

```diff
@@ -1830,6 +1916,12 @@ watch_child(char *argv[])
 
     int nullfd;
 
+    // XXX: zero values are not supported because they result in
+    // misconfigured SMP Squid instances running forever, endlessly
+    // restarting each dying kid.
+    if (Config.hopelessKidRevivalDelay == 0)
```

Comment:
Please use `<= 0` to better match the error message (even if some other code is already testing that configured time_t values cannot be negative).